meson.build

```diff
@@ -48,10 +48,45 @@ targets = []
 
 cc = meson.get_compiler('c')
 fs = import('fs')
+
+# Find the picolibc name for the host cpu. Provide
+# for some common aliases
+
+cpu_family_aliases = {
+		       # aarch64
+		       'arm64' : 'aarch64',
+		       # cris
+		       'crisv32' : 'cris',
+		       # i386
+		       'i486' : 'i386',
+		       'i586' : 'i386',
+		       'i686' : 'i386',
```

Comment:
Meson will never return these ix86 as a cpu family, but as we know we can set it in a cross file.  Is that why we have them?
https://mesonbuild.com/Reference-tables.html#cpu-families

Reply:
Hmm... it appears that meson doesn't like it and warns about it while setup.
```
$ ./scripts/do-i386-configure builddir-i386 
WARNING: Unknown CPU family i386, please report this at https://github.com/mesonbuild/meson/issues/new
```